server/db/models/wine.js

```diff
@@ -1,7 +1,9 @@
 var mongoose = require('mongoose');
+var Review = require('./review');
```

Comment:
since you used mongoose.model() to create the model, you don't need to save the value of this require